### PR TITLE
Fix fixture imports for latest Prefect version

### PR DESCRIPTION
**Pull request by madkinsz**

### CODE REVIEW

**Comment by madkinsz** [comment]
Should be resolved on release of Prefect today

File: prefect_dask/task_runners.py [hunk 1]
```diff
@@ -268,7 +268,11 @@ def visit_fn(expr):
     async def wait(self, key: UUID, timeout: float = None) -> Optional[State]:
         future = self._get_dask_future(key)
         try:
-            return await future.result(timeout=timeout)
+            result = future.result(timeout=timeout)
+            if inspect.isawaitable(result):
+                return await result
+            else:
+                return result
```
Comment (madkinsz) [inline prefect_dask/task_runners.py:275]: No this isn't safe. We must be awaiting the retrieval of a result from a future. If it's not returning a coroutine we're blocking the event loop.

File: tests/test_utils.py [hunk 2]
```diff
@@ -16,15 +16,15 @@ def test_task():
                 result = client.compute(delayed_num).result()
             return result
 
-        @flow(task_runner=DaskTaskRunner)
+        @flow(task_runner=DaskTaskRunner())
```
Comment (madkinsz) [inline tests/test_utils.py:19]: Why these changes?

File: prefect_dask/task_runners.py (hunk 1 above)
Comment (desertaxle) [reply prefect_dask/task_runners.py]: Gotcha, reverted this change.

File: tests/test_utils.py (hunk 2 above)
Comment (desertaxle) [reply tests/test_utils.py]: This was based on the type hints, but I see this is incorrect. I've reverted these changes.

File: prefect_dask/task_runners.py [hunk 3]
```diff
@@ -268,11 +268,7 @@ def visit_fn(expr):
     async def wait(self, key: UUID, timeout: float = None) -> Optional[State]:
         future = self._get_dask_future(key)
         try:
-            result = future.result(timeout=timeout)
-            if inspect.isawaitable(result):
-                return await result
-            else:
-                return result
+            await future.result(timeout=timeout)
```
Comment (madkinsz) [inline prefect_dask/task_runners.py]: Missing the return now :)

Comment (desertaxle) [reply prefect_dask/task_runners.py]: 🤦

**Comment by madkinsz** [comment]
Good to see the standards test suite works :D 

Work moving over to #93 I guess since both will be needed for merge.
